jupyterlab_git/handlers.py

```diff
@@ -534,6 +534,34 @@ async def post(self):
         self.finish(json.dumps(response))
 
 
+class GitIgnoreHandler(GitHandler):
+    """
+    Handler to manage .gitignore
+    """
+
+    @web.authenticated
+    async def post(self):
+        """
+        POST add entry in .gitignore
+        """
+        data = self.get_json_body()
+        top_repo_path = data["top_repo_path"]
+        file_path = data.get("file_path", None)
+        use_extension = data.get("use_extension", False)
+        if file_path:
+            if use_extension:
+                parts = os.path.splitext(file_path)
+                if len(parts) == 2:
+                    file_path = "**/*" + parts[1]
```

Comment:
For file extension like `.tar.bz2`, `parts` will be `("name.tar", ".bz2")`. Is ignoring `.bz2` what is expected or is it ignoring `.tar.bz2`?

If this is the second option, I advice using again `pathlib.Path`. So you could request the full list of extensions from the `suffixes` property.

```py
pathlib.Path(file_path).suffixes
```

Reply:
Thx for the review. Last commit implements this.